bigbluebutton-html5/imports/ui/components/nav-bar/service.js

```diff
@@ -10,11 +10,7 @@ const getBreakoutJoinURL = (breakout) => {
     const user = breakout.users.find(u => u.userId === currentUserId);
 
     if (user) {
-      const { urlParams } = user;
-      return [
-        window.origin,
-        `html5client/join?sessionToken=${urlParams.sessionToken}`,
-      ].join('/');
+      return user.redirectToHtml5JoinURL;
```

Comment:
Are you sure you dont need the `window.origin` here?

Reply:
redirectToHtml5JoinURL: 'https://anton3.blindside-dev.com/bigbluebutton/api/join?fullName=b&isBreakout=true&joinViaHtml5=true&meetingID=506cc48fb1940cc3ea92c8d2c279286528e3ea6e-1528387877791&password=mp&redirect=true&userID=w_ii3jukcc1ptr-1&checksum=32d4f1974716efa684df8e45a6da65cb90654c78' -- it's working fine without